config/overlays/e2e-release-0.3/kustomization.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: baremetal-operator-system
+resources:
+  - https://github.com/metal3-io/baremetal-operator/config/tls?ref=release-0.3
```

Comment:
Please use consistent k8s formatting, ie. without leading spaces for the lists. You have both now in this file.

Reply:
Thanks. I've fixed it.